spec/requests/api/v1/docs/followed_tags_spec.rb

```diff
@@ -0,0 +1,50 @@
+require "rails_helper"
+require "swagger_helper"
+
+# rubocop:disable RSpec/EmptyExampleGroup
+# rubocop:disable RSpec/VariableName
+
+RSpec.describe "Api::V1::Docs::FollowedTags" do
+  let(:Accept) { "application/vnd.forem.api-v1+json" }
+  let(:api_secret) { create(:api_secret) }
+  let(:user) { api_secret.user }
+  let(:tag1) { create(:tag) }
+  let(:tag2) { create(:tag) }
+
+  before do
+    user.follow(tag1)
+    user.follow(tag2)
+  end
```

Comment:
@lboogie04 you're setting up some seed tags here but there's some information missing, hence you'll notice you're missing the example values in the response. 

<img width="728" alt="Screenshot 2023-01-17 at 09 04 16" src="https://user-images.githubusercontent.com/2786819/212832224-dc6a82b1-4ebc-4f34-8745-3ba871aeac0c.png">

I've added some suggestions to get the seed data to work and show up in the response. 

```suggestion
  let(:tag1) { create(:tag) }
  let(:tag2) { create(:tag) }

  let(:tag1_json) { { id: tag1.id, name: tag1.name, points: 1.0 } }
  let(:tag2_json) { { id: tag2.id, name: tag2.name, points: 1.0 } }

  before do
    [tag1, tag2].each { |tag| user.follow(tag) }
  end
```

If you apply the suggestion and re-generate the JSON file using `SWAGGER_DRY_RUN=0 RAILS_ENV=test rails rswag PATTERN="spec/requests/api/v1/**/*_spec.rb"` you'll notice that the example value will now show up:

<img width="731" alt="Screenshot 2023-01-17 at 09 05 11" src="https://user-images.githubusercontent.com/2786819/212832309-4dece165-1900-4c22-a760-19473aadb291.png">

You'll see a similar setup in [v0's documentation](https://github.com/forem/forem/blob/19fa8df0f4729afcc049397752b9a140b21b52c3/spec/requests/api/v0/follows_spec.rb#L45-L52).

Hope this is helpful. Let me know if you have any follow up questions.